_docs-sources/pipelines/core-concepts/index.md

```diff
@@ -0,0 +1,27 @@
+# Core concepts
+
+## Overview
+
+Fundamentally, Gruntwork Pipelines is designed to roll out an _infrastructure change._ Doing that in a secure and responsible way requires understanding a few key concepts.
+
+## Concepts
+
+### Infrastructure change
+
+When you edit any infrastructure-as-code that needs to be "applied" to your cloud account (e.g. AWS or GCP), you are making an _infrastructure change_. We sometime call these an "infra-change" for short.
+
+Infra-changes can involve updates to OpenTF (Terraform) code, Packer templates, Docker files, Kubernetes Helm charts, or any other type of file that represents a desired state of your infrastructure and that needs to be somehow applied. A classic example is changing some variables on an instance of a Terraform module. By changing these variables you are essentially requesting that some API calls eventually be made to AWS to update your infrastructure in some way (in this case, by running `terraform apply` at some point).
+
+Gruntwork Pipelines assumes that infra-changes are committed via git, usually by first opening a Pull Request. When you open a Pull Request, you are essentially "proposing" an infra-change.
+
+### Infrastructure change set
+
+Sometimes users create a Pull Request that changes more than one file at a time. And sometimes a change to a single file affects how multiple other files will be "applied" to your live cloud account. For example, many users of Terragrunt use a pattern known as "envcommon" as a way to specify a default set of values for modules.
```

Comment:
```suggestion
An infrastructure change set (or infra-change set) is a collection of infra-changes determined by the changes to your IaC code.

There are two ways an infra-change set is generated -  Users creating a Pull Request that changes one or more environment specific files and when a default module is updated when using the [module_default pattern](../link-to-page). Changes to environment specific files result in an action running for that specific infrastructure unit. Changes to default modules result in actions being run in all environments that reference that default module.
```

I think we need to re-word this section. We don't explain what an infra change set is until we explain the types of changes that can be made, I think that's a bit backwards.

Reply:
Roger; feel free to take the lead.